dgraph/cmd/zero/zero_test.go

```diff
@@ -112,3 +118,87 @@ func TestProposalKey(t *testing.T) {
 	}
 	require.Equal(t, len(uniqueKeys), 10, "each iteration should create unique key")
 }
+
+func validateJsonHealthContent(body []byte) error {
+	var r map[string]interface{}
+	e := json.Unmarshal(body, &r)
+	if e != nil {
```

Comment:
we should take `t` of type `*testing.T` as an arg to this function so that we can reduce the boiler plate code here. Once you have access to `t`, you could just do `require.NoError(t, err)`